docs/source/developers/java/building.rst

```diff
@@ -107,33 +111,48 @@ Maven
         $ cd arrow/java
         $ export JAVA_HOME=<absolute path to your java home>
         $ java --version
-        $ mvn clean generate-resources -Pgenerate-cdata-dylib_so -N
+        $ mvn generate-resources -Pgenerate-libs-cdata-all-os -N
+        $ # MacOS / Linux
         $ ls -latr ../java-dist/lib
         |__ libarrow_cdata_jni.dylib
+        |__ libarrow_cdata_jni.so
+        $ # Windows
```

Comment:
For consistency and clarity, maybe separate MacOS/Linux and Windows into different blocks under different headings as you did below

Reply:
Changed